File: LLM-Translations/GPT-4/codenet/Go/C/s054878611.c
#include <stdio.h>

int solution(int n, int a[]) {
    int memo[100002] = {0};
    for (int i = 0; i < n; i++) {
        int num = a[i];
        memo[num]++;
        memo[num+1]++;
        memo[num+2]++;
    }
    int out = 0;
    for (int i = 0; i < 100002; i++) {
        if (memo[i] > 0) {
            if (memo[i] > out) {
                out = memo[i];
            }
        }
    }
    return out;
}

int main() {
    int n;
    scanf("%d", &n);
    int a[n];
    for (int i = 0; i < n; i++) {
        scanf("%d", &a[i]);
    }
    printf("%d\n", solution(n, a));
    return 0;
}
// 

File: LLM-Translations/GPT-4/codenet/Go/Python/s054878611.py
def solution(n: int, a: List[int]) -> int:
    memo = [0] * (int(1e5) + 2)
    for num in a:
        memo[num] += 1
        memo[num + 1] += 1
        memo[num + 2] += 1
    out = 0
    for num in memo:
        if num > 0:
            out = max(out, num)
    return out

def max(x: int, y: int) -> int:
    if x > y:
        return x
    return y

if __name__ == '__main__':
    n = int(input())
    a = list(map(int, input().split()))
    print(solution(n, a))


